### ISSUE TITLE
Use different permissions when mounting archive.

### ISSUE THREAD

**Issue by geckolinux** [borgbase/vorta#520]
Hi, I'm running Vorta 0.6.26 and Borg 1.1.13 on openSUSE Tumbleweed.

I just discovered a really weird anomaly with Vorta. I tested a backup of my Firefox `~/.mozilla` directory by using Vorta to mount the repo to `~/Desktop/a`, renaming my `~/.mozilla` folder and using Dolphin to copy it from `~/Desktop/a/home/myself/.mozilla` to its normal location. This consistently led to a corrupt Firefox profile, as described [here](https://support.mozilla.org/en-US/kb/fix-bookmarks-and-history-will-not-be-functional). I don't understand all the moving parts in Firefox, but it's either a permissions problem or some other kind of corruption with one of the sqlite databases it uses.

By simply creating a filesystem copy of the `~/.mozilla` profile on my local disk and closing Firefox and renaming the original profile and copying back the backup, Firefox launches as expected as if nothing happened.

So I tested again with the same process as described above, except I manually created the backup and manually mounted it with `borg create` and `borg mount`, and repeated the profile restore process. This also worked as expected.

Then I manually mounted with `borg mount` a backup created with Vorta and restored my profile from that, which also worked correctly.

Then I used Vorta to mount the manually created borg backup described above, and this gave me a corrupted profile.

So Vorta must be doing something weird with the way it's mounting the backups. I confirmed this happens every time I tried to mount it via Vorta and restore the profile, both from a USB HD repo and from my NAS over SSH.

Thanks for the help.

**Comment by samuel-w** [comment]
I believe this is caused by permissions, specifically the "-o umask=0277,uid=1000" portion of the mount command vorta uses. I ran the mount command with the options, and all the files became read only. I then ran the command without the options, and all the files kept their permissions.

Update: I narrowed it down to the umask=0277 portion, running mount as 
`borg --log-json mount -o uid=1000 SOURCE DESTINATION`
seems to preserve permissions.
